kernel-headers/rdma/vmw_pvrdma-abi.h

```diff
@@ -49,7 +49,11 @@
 
 #include <linux/types.h>
 
-#define PVRDMA_UVERBS_ABI_VERSION	3		/* ABI Version. */
```

Comment:
When you resubmit this, kernel headers have to be in their own commit

Reply:
@jgunthorpe Does this apply to the kernel changes too?